app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -93,10 +94,24 @@
     private int chartZoom = 3;
     private boolean hasZoomedChart = false;
 
-    private NumberFormat sgvFormatter;
-    private boolean mmolxl;
+    public static NumberFormat sgvFormatter;
+    public static boolean mmolxl;
     private boolean mmolxlDecimals;
 
+    public static byte radioChannelInuse = 0;
```

Comment:
These public statics seem like they should be privately contained in the classes that use them.